tests/realtikvtest/importintotest4/cloud_sdk_test.go

```diff
@@ -249,6 +258,15 @@ func (s *mockGCSSuite) TestAutoDetectFileType() {
 		}
 		err := s.tk.QueryToErr(badImportSQL)
 		s.Require().ErrorContains(err, nc.wantSubstr)
+		s.T().Logf("the task id is %d", storage.TestLastTaskID.Load())
+		// wait cleanup done, so the table mode is switched back to normal
+		// Note: the first case doesn't submit the task, so no task id, but it's
+		// ok to call GetTaskByID with non-existing id, it will just return
+		// ErrTaskNotFound immediately.
+		s.Eventually(func() bool {
+			_, err2 := taskMgr.GetTaskByID(ctx, storage.TestLastTaskID.Load())
+			return goerrors.Is(err2, storage.ErrTaskNotFound)
+		}, 30*time.Second, 100*time.Millisecond)
```

Comment:
Is this duration(30*time.Second) too long? In the extreme case, the entire CI test directly reports a timeout without reporting any errors for this test, resulting in uncertainty about which test has exceeded the time limit and reported an error.

Reply:
I just think 30 seconds is quite a long time. 
Additionally, this package bazel has set a time limit of 300 seconds. If multiple tests reach 30 seconds, it could easily cause the entire CI test to timeout. I feel that what we are talking about is not a timeout issue.
However, such tests that take more than 30 seconds might be rather low in quality.